internal/v4/relations.go

```diff
@@ -134,8 +135,12 @@ func (h *Handler) getRelatedIfaceResponses(
 					return nil, err
 				}
 				// Build the response.
+				url := entity.URL
+				if entity.PromulgatedURL != nil {
```

Comment:
I had to think quite hard about this. Perhaps a comment might be justified:

    // Always use a promulgated URL if one is available.

?